### PR TITLE
Map open detail view from collection view

### PR DESCRIPTION
**Pull request by kristen**
#111

### CODE REVIEW

File: Mapstagram/ViewControllers/MapViewController.swift [hunk 1]
```diff
@@ -237,47 +231,51 @@ extension MapViewController: UICollectionViewDataSource {
 // MARK: - CollectionViewDelegate
 extension MapViewController: UICollectionViewDelegate {
     func collectionView(collectionView: UICollectionView, didSelectItemAtIndexPath indexPath: NSIndexPath) {
-        if let lastSelectedPhotoCell = lastSelectedPhotoCell {
-            lastSelectedPhotoCell.setSelected(false)
+        let media = SessionService.getInstance().getMediaArray()[indexPath.row]
+        
+        if lastSelectedPhotoCellIndexPath != nil && lastSelectedPhotoCellIndexPath! == indexPath {
+            let detailViewController = DetailViewController(media: media)
```
Comment (jamietsao) [inline Mapstagram/ViewControllers/MapViewController.swift]: :+1:

File: Mapstagram/Views/SelectableGridCell.swift [hunk 2]
```diff
@@ -33,15 +33,17 @@ class SelectableGridCell: UICollectionViewCell {
...
     
-    func setSelected(selected: Bool) {
-        UIView.animateWithDuration(0.3, animations: {
-            if selected {
-                self.selectedOverlayView.alpha = 0.4
-                self.selectedOverlayBorderViews.map { $0.alpha = 1 }
-            } else {
-                self.selectedOverlayView.alpha = 0
-                self.selectedOverlayBorderViews.map { $0.alpha = 0 }
+    override var selected: Bool {
```
Comment (jamietsao) [inline Mapstagram/Views/SelectableGridCell.swift]: :+1:

**Comment by jamietsao** [comment]
Looks great.  Merge it!

File: Mapstagram/Views/SelectableGridCell.swift (hunk 2 above)
Comment (kristen) [inline Mapstagram/Views/SelectableGridCell.swift]: Pretty sweet right? UICollectionViewCells have their own property of selected that gets set automatically when appropriate in the UICollectionView!

**Pull request merged**